lia-ui/src/main/java/com/lithium/community/android/ui/components/activities/LiCreateMessageActivity.java

```diff
@@ -144,7 +145,13 @@ protected void onCreate(Bundle savedInstanceState) {
             fragmentBundle.putAll(actBundle);
         }
         fragment = LiCreateMessageFragment.newInstance(fragmentBundle);
-        ft.replace(R.id.li_create_message_fragment_container, fragment, fragment.getClass().getName());
+        Fragment f = fm.findFragmentById(R.id.li_create_message_fragment_container);
+        if(f instanceof LiCreateMessageFragment){
+            fragment = (LiCreateMessageFragment) f;
+            f.setArguments(fragmentBundle);
+        }else {
+            ft.replace(R.id.li_create_message_fragment_container, fragment, fragment.getClass().getName());
```

Comment:
If the fragment is `null` then `ft.replace` will throw an NPE. Can we implement it a bit more defensively?

Reply:
this will be unlikely, newInstance returns unfailingly.